### PR TITLE
lambda-term 1.9

### PR DESCRIPTION
**Pull request by diml**

### CODE REVIEW

File: packages/lambda-term/lambda-term.1.9/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/diml/lambda-term/archive/1.8.tar.gz"
```
Comment (dsheets) [inline packages/lambda-term/lambda-term.1.9/url]: This looks wrong.

**Comment by samoht** [comment]
Doesn't work on 4.00:

```
===== ERROR while installing lambda-term.1.9 =====
# opam-version 1.1.1
# os           linux
# command      ocaml setup.ml -build
# path         /home/travis/.opam/system/build/lambda-term.1.9
# compiler     system (4.00.1)
# exit-code    1
# env-file     /home/travis/.opam/system/build/lambda-term.1.9/lambda-term-14251-ad6566.env
# stdout-file  /home/travis/.opam/system/build/lambda-term.1.9/lambda-term-14251-ad6566.out
# stderr-file  /home/travis/.opam/system/build/lambda-term.1.9/lambda-term-14251-ad6566.err
### stdout ###
# ...[truncated]
# /usr/bin/ocamllex.opt -q src/lTerm_inputrc.mll
# /home/travis/.opam/system/bin/ocamlfind ocamldep -package lwt -package lwt.react -package lwt.unix -package zed -modules src/lTerm_inputrc.ml > src/lTerm_inputrc.ml.depends
# /home/travis/.opam/system/bin/ocamlfind ocamldep -package lwt -package lwt.react -package lwt.unix -package zed -modules src/lTerm_history.mli > src/lTerm_history.mli.depends
# /home/travis/.opam/system/bin/ocamlfind ocamlc -c -g -annot -ccopt -I/home/travis/.opam/system/lib/lwt -package lwt -package lwt.react -package lwt.unix -package zed -I src -I src/widget_impl -o src/lTerm_history.cmi src/lTerm_history.mli
# /home/travis/.opam/system/bin/ocamlfind ocamldep -package lwt -package lwt.react -package lwt.unix -package zed -modules src/lTerm_history.ml > src/lTerm_history.ml.depends
# /home/travis/.opam/system/bin/ocamlfind ocamlc -c -g -annot -ccopt -I/home/travis/.opam/system/lib/lwt -package lwt -package lwt.react -package lwt.unix -package zed -I src -I src/widget_impl -o src/lTerm.cmo src/lTerm.ml
# + /home/travis/.opam/system/bin/ocamlfind ocamlc -c -g -annot -ccopt -I/home/travis/.opam/system/lib/lwt -package lwt -package lwt.react -package lwt.unix -package zed -I src -I src/widget_impl -o src/lTerm.cmo src/lTerm.ml
# File "src/lTerm.ml", line 171, characters 16-25:
# Error: Unbound value Sys.win32
# Command exited with code 2.
### stderr ###
```

**Comment by yallop** [comment]
Hmm. It has

```
available: [ "ocaml-version" >= "4.01" ]
```

File: packages/lambda-term/lambda-term.1.9/opam [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+install: [["ocaml" "setup.ml" "-install"]]
+build-doc: [["ocaml" "setup.ml" "-doc"]]
+remove: [["ocamlfind" "remove" "lambda-term"]]
+depends: [
+  "ocamlfind"
+  "lwt" {>= "2.4.0"}
+  "zed" {>= "1.2"}
+  "react" {>= "1.0.0"}
+]
+available: [ "ocaml-version" >= "4.01" ]
```
Comment (samoht) [inline packages/lambda-term/lambda-term.1.9/opam]: should be `ocaml-version` (without quotes)

**Comment by diml** [comment]
I fixed the contraint

**Comment by diml** [comment]
The last build failure is due to a download error

**Comment by samoht** [comment]
I restarted the job

**Pull request merged**